examples/spouts/go-rabbitmq-spout/source/go.mod

```diff
@@ -186,3 +154,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+// until the changes in github.com/pachyderm/dex are upstreamed to github.com/dexidp/dex, we swap in our repo
+replace github.com/dexidp/dex => github.com/pachyderm/dex v0.0.0-20230426001747-706aec218aba
```

Comment:
Is this actually needed in the rabbitmq example?

Reply:
I got repeated errors locally about our version of dexidp until I added it. It was the last thing I added, then I committed … and now it seems fine with it. Wish I had a log, but it fell off the end of my scrollback buffer. I’ve doubled its length for the future. Very odd.